packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -140,4 +164,27 @@ export default class QuanticResultQuickview extends LightningElement {
   get buttonLabel() {
     return this.hasNoPreview ? this.labels.noPreview : this.labels.openPreview;
   }
+
+  get buttonClass() {
+    let classNames = ['slds-button'];
+
+    if (!this.previewButtonVariant || !this.previewButtonLabel) {
```

Comment:
I think there is an issue with the condition here. Suppose the component is set like that:

    <c-quantic-result-quickview preview-button-variant="brand" preview-button-label="" ...>

As `previewButtonVariant` is truthy and `previewButtonLabel` is falsy, we'll enter the `if`, and the class `slds-button_brand` would not be added to the button.

Reply:
Actually, I made this on purpose, to avoid having buttons  like this : 

<img width="569" alt="outline" src="https://user-images.githubusercontent.com/86681870/145465112-0f6f8937-c0f1-4b8b-b1ba-9483abceaeea.png">

or this  :

<img width="596" alt="brandEmpty" src="https://user-images.githubusercontent.com/86681870/145465432-c42739a6-f352-418c-a28f-caa54c4ea824.png">


So I made this in  way that when there is no label and we will have just a button containing an icon I force the button to be in the base variant  to have always something like this: 

<img width="579" alt="normalDefault" src="https://user-images.githubusercontent.com/86681870/145465740-6d265b23-59a4-4bbc-a001-27314cdef890.png">

@lbergeron,  does this make sense or we need to allow having any variant even when there is no variant ?

Reply:
Personally, I prefer to apply the variant when it is set.